lib/jekyll/tags/strapiimagefilter.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+require "down"
+
+module Jekyll
+    module StrapiImageFilter
+      def asset_url(input)
+        strapi_endpoint = @context.registers[:site].config['strapi']['endpoint']
+        uri_path = "#{strapi_endpoint}#{input['url']}"
+        if not Dir.exist?('assets')
```

Comment:
I use my **assets** directory for website stuff and I use my **uploads** folder for my post and product images.

Reply:
Hello! Did the files appear in assets directory?

I checked this part now, and it could be that Jekyll is copying assets before executing filter. 

I will look into one of solutions:
* Copy extra assets after the filters running
* Copy them directly to the expected output _site/... instead being dependent on the build process and its flow. But this sounds a bit against a design, so let me think. 

Thanks!

Reply:
For the files to appear, I have to add a new content type called photos...maybe that is why it isn't downloading from assets. I will create that content type now.

Reply:
The images didn't download for me. I checked both folders: /assets and _site/assets and _site/assets/images.

I did receive an error because photos content type was private.

Reply:
The api is working and populating the json data for photos via https://api.example.com/api/photos?populate=*

On the homepage, all I see is the name Title..earlier I forgot to **publish** the photo.